docs/integrations/kafka/kafka-vector.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_label: Vector
+sidebar_position: 7
+description: Using Vector with Kafka and ClickHouse
+---
+
+# Using Vector with Kafka and ClickHouse
+
+ Vector is a vendor-agnostic data pipeline with the ability to read from Kafka and send events to ClickHouse.
+
+ A [getting started](../vector-to-clickhouse) guide for Vector with ClickHouse focuses on the log use case and reading events from a file. We utilize the [Github sample dataset](https://datasets-documentation.s3.eu-west-3.amazonaws.com/kafka/github_all_columns.ndjson) with events held on a Kafka topic.
+
+Vector utilizes [sources](https://vector.dev/docs/about/concepts/#sources) for retrieving data through a push or pull model. [Sinks](https://vector.dev/docs/about/concepts/#sinks) meanwhile provide a destination for events. We, therefore, utilize the Kafka source and ClickHouse sink. Note that whilst Kafka is supported as a Sink, a ClickHouse source is not available. Vector is as a result not appropriate for users wishing to transfer data to Kafka from ClickHouse.
+
+ Vector also supports the [transformation](https://vector.dev/docs/reference/configuration/transforms/) of data. This is beyond the scope of this guide. The user is referred to the Vector documentation should they need this on their dataset.
+
+ Note that the current implementation of the ClickHouse sink utilizes the HTTP interface. The ClickHouse source does not support the use of a JSON schema at this time. Data must be published to Kafka in either plain JSON format or as Strings.
```

Comment:
> The ClickHouse source does not support the use of a JSON schema at this time. Data must be published to Kafka in either plain JSON format or as Strings.

What do we mean by the `ClickHouse source` in that context ? should it be `sink` or is it during the initial dataset generation? if yes, then we can leave out this bit as the reader can be confused by the "source" terminology also used by vector

Reply:
i mean sink